component/ldap.libsonnet

```diff
@@ -1,13 +1,13 @@
+local common = import 'common.libjsonnet';
```

Comment:
In this file, line 23, you need to do something like

```
    bindPassword:
      if std.isString(idp.ldap.bindPassword) then
        idp.ldap.bindPassword
      else
        params.secrets[idp.ldap.bindPassword.name].bindPassword,
```

assuming you go with my suggestion which doesn't need `bindPasswordSecretRef`.